package org.testing_survey_creator.exception

class CustomAuthException(message: String, cause: Throwable) : RuntimeException(message, cause)